include/SFML/Window/ContextSettings.hpp

```diff
@@ -26,6 +26,7 @@
 #define SFML_CONTEXTSETTINGS_HPP
 
 #include <SFML/Config.hpp>
+#include <tuple>
```

Comment:
`<tuple>` is a bit of an expensive header, which can increase compilation times a lot. I would prefer to not use it at all, or to (more simply) hide it in `ContextSettings.cpp`, so that it doesn't get virally included by users of `ContextSettings.hpp`.